src/includes/getting-started-primer/flutter.mdx

```diff
@@ -1,7 +1,18 @@
 <Note>
 
-_Sentry's Flutter SDK enables capturing sessions for Release Health, offline caching as well as reporting messages and errors._
-
-_Sentry's Flutter SDK depends on the [Dart SDK](/platforms/dart/) and includes support to native crashes through Sentry's native SDKs (Android and iOS), It'll capture errors in the native layer, including Java, Kotlin, C and C++ code for `Android` and ObjC, Swift and C for `iOS`._
+_Sentry's Flutter SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/)._
 
 </Note>
+
+Features:
+
+- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS](/platforms/apple/)), including
```

Comment:
we also Support Flutter Web, it's a bit limited compared to Android and iOS, of course.
Eg, events are not enriched with device context.

Reply:
I've added "Limited Support for Flutter Web" - LMK if there's a link to add or if that wording is quite what we want here @bruno-garcia

Reply:
sounds good to me, so we don't create much expectations from Web devs as we didn't spend time on it